pinc/DPDatabase.inc

```diff
@@ -82,6 +82,11 @@ final class DPDatabase
         return mysqli_real_escape_string(self::$_connection, $value);
     }
 
+    public static function escape_like_wildcards($value)
+    {
+        return addcslashes($value, "%_[]^-");
```

Comment:
According to https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html#operator_like it's only `_` and `%` as `LIKE` wildcards. The other looks like they're for regex-based patterns which I wouldn't include here.

Reply:
Thanks - removed superfluous wildcards from new `escape_like_wildcards` function